js/components/graph/EdgeGroup.js

```diff
@@ -7,10 +7,6 @@ import Edge from "./Edge"
 export default class EdgeGroup extends React.Component {
   constructor(props) {
     super(props)
-    // EdgeGroup's state is used to keep track of the edgeIDs of
```

Comment:
This is good, but now that you've removed all of the state there's no reason to override the constructor at all, so you can just delete this entire method.